<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="YButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>105</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>YES</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="NButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>105</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>NO</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="R0Button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>105</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>R0</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="R1Button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>105</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>R1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="R2Button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>105</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>R2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="R3Button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>105</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>R3</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>461</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QRadioButton" name="radioButton_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>105</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>G0</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>105</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>G2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_9">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>105</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>G1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_10">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>105</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>G3</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>GO!!GO!!GO!!</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="TriBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>89</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Tri</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>89</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>89</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>89</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>89</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
